config/sync/registration_validation.settings.yml

```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "*admin*\r\nroot\r\n*master*"
```

Comment:
Why did you remove the other user names that used to be here?

Reply:
because they contain the wildcard words and therefore were redundant.

Reply:
"rootuser" is a selectable username, please also turn root into a wildcard.

Reply:
done